rpc/src/rpc_service.rs

```diff
@@ -84,6 +84,7 @@ struct RpcRequestMiddleware {
     /// Initialized based on vote_accounts_to_monitor, maps identity
     /// pubkey associated with the vote account to the validator info.
     identity_info_map: Arc<IdentityInfoMap>,
+    enable_prometheus_metrics: Arc<AtomicBool>,
```

Comment:
```suggestion
    enable_prometheus_metrics: bool,
```

I think we can get away with a regular bool here, the value does not change at runtime, and unlike e.g. `vote_accounts_to_monitor` it is small, so it makes sense to just copy the bool so every thread has its own bool. (For `vote_accounts_to_monitor`, the value also does not change at runtime, but the hash map is larger than a few pointers, so we can share a single instance among all threads by putting it in an `Arc`. But the bool is a single byte, the `Arc` is much larger than the bool itself.)

Reply:
done